gradoop-flink/src/main/java/org/gradoop/flink/model/api/tpgm/TemporalGraphOperators.java

```diff
@@ -42,6 +50,107 @@
   // Unary Operators
   //----------------------------------------------------------------------------
 
+  /**
+   * Extracts a snapshot of a temporal graph using a given temporal predicate. This will calculate
```

Comment:
Maybe write
> of this temporal graph

instead to make it clear that this graph if transformed (although that should be clear).